modules/EvseV2G/crypto/crypto_openssl.cpp

```diff
@@ -129,17 +129,18 @@ bool check_iso2_signature(const struct iso2_SignatureType* iso2_signature, EVP_P
 
 bool load_contract_root_cert(::openssl::certificate_list& trust_anchors, const char* V2G_file_path,
                              const char* MO_file_path) {
-    // note the file(s) may contain more than one certificate (hence must be PEM)
-    // try MO_file_path first then fallback to V2G_file_path
-
     trust_anchors.clear();
-    trust_anchors = ::openssl::load_certificates(MO_file_path);
+
+    auto mo_certs = ::openssl::load_certificates(MO_file_path);
+    trust_anchors.insert(trust_anchors.end(), std::make_move_iterator(mo_certs.begin()),
+                         std::make_move_iterator(mo_certs.end()));
+
+    auto v2g_certs = ::openssl::load_certificates(V2G_file_path);
+    trust_anchors.insert(trust_anchors.end(), std::make_move_iterator(v2g_certs.begin()),
+                         std::make_move_iterator(v2g_certs.end()));
```

Comment:
@james-ctc @SebaLukas , according to hubject we can always use v2g and mo as trust anchors. I have not checked if this is in line with iso15118

Reply:
What does Hubject mean by that? The contract leaf and the sub CA 1 and 2 certificates can be based on the MO or on the V2G Root. That is why we load both roots here. If one is not available, the other can still be used.

Reply:
Ah, now I understand. Before the code change it was either or. Then Hubject is right. If there are several roots (MO and V2G), they should all be loaded. Not just MO or V2G